app/src/main/java/com/duckduckgo/app/systemsearch/SystemSearchViewModel.kt

```diff
@@ -359,6 +366,12 @@ class SystemSearchViewModel @Inject constructor(
         }
     }
 
+    fun deleteSavedSiteSnackbarDismissed(savedSite: SavedSite) {
+        appCoroutineScope.launch(dispatchers.io()) {
+            savedSitesRepository.delete(savedSite, true)
```

Comment:
why do you need to delete the bookmark here? isn’t it just a favourite?

Reply:
No, that's `deleteFavoriteSnackbarDismissed` which only removes the favorite.
This needs to delete the favorite's associated bookmark.